include/autoppl/expression/variable/constant.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+namespace ppl {
+namespace expr {
+
+template <class ValueType>
+struct Constant
```

Comment:
What is Constant for?

Reply:
All the distribution objects are constrained to only hold parameters that are "variable expressions". A literal is not a variable expression. I made a light-weight wrapper of such constants that does fit the concept of "variable expression". Another reason why having these separate `uniform` and `normal` functions because these guys can take in any types, convert them however way we want and then pass them into these classes.